package-lock.json

```diff
@@ -64,7 +64,7 @@
         "chai-arrays": "2.2.0",
         "chai-as-promised": "7.1.1",
         "cheerio": "1.0.0-rc.12",
-        "chokidar-cli": "*",
```

Comment:
I think this should be excluded unless we’re actually trying to update dependencies.

Reply:
any reason why we can't update chokidar? I see this change every time I `npm i` so it'd be nice to merge this.

Reply:
I've reverted this back to `latest` for the time being as I guess it's not part of the story?